webapp/components/admin_console/storage_settings.jsx

```diff
@@ -241,6 +243,25 @@ export default class StorageSettings extends AdminSettings {
                     disabled={this.state.driverName !== DRIVER_S3}
                 />
                 <TextSetting
+                    id='amazonS3Region'
+                    label={
+                        <FormattedMessage
+                            id='admin.image.amazonS3RegionTitle'
+                            defaultMessage='Amazon S3 Region:'
+                        />
+                    }
+                    placeholder={Utils.localizeMessage('admin.image.amazonS3RegionExample', 'Ex "us-east-1"')}
+                    helpText={
+                        <FormattedMessage
+                            id='admin.image.amazonS3RegionDescription'
+                            defaultMessage='(Optional) AWS region you selected when creating your S3 bucket. If no region is set, Mattermost attempts to get the appropriate region from AWS, or sets it to "us-east-1" if none found.'
```

Comment:
In what case would you not be able to get the region? I don't know about Minio, but with AWS, there is no circumstance where you would be unable to get the region for a bucket.

Reply:
I agree, and that should be one of the required fields when enabling S3 storage. Might be better to discuss in UX/PM Channel as this is specified in the Jira ticket.